pkg/pod/entrypoint_lookup_impl.go

```diff
@@ -139,6 +139,11 @@ func buildCommandMap(idx v1.ImageIndex, hasArgs bool) (map[string][]string, erro
 	}
 	for _, desc := range mf.Manifests {
 		plat := desc.Platform.String()
+		// skip unknown platforms.
+		// Docker uses these to store attestation data: https://docs.docker.com/build/attestations/attestation-storage/#examples
+		if plat == "unknown/unknown" {
```

Comment:
I am curious if it is guaranteed whenever the OS is `unknown` the Architecture will always be `unknown` or is it possible to have one of them defined and the other `unkown`.

Reply:
This is already included in the [link](https://docs.docker.com/build/attestations/attestation-storage) of the note. it says:：`To prevent container runtimes from accidentally pulling or running the image described in the manifest, the platform property of the attestation manifest will be set to unknown/unknown`